aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/plugins/ElasticBeanstalkPlugin.java

```diff
@@ -62,6 +64,12 @@ public boolean isEnabled() {
 
     public void populateRuntimeContext() {
         byte[] manifestBytes = new byte[0];
+
+        // lazily initialize for performance on unused plugins
+        if (OBJECT_MAPPER == null) {
+            OBJECT_MAPPER = new ObjectMapper();
```

Comment:
Since this is only called once anyways I guess there's no point in even having a `static` variable if we're lazying

Reply:
Ah sorry I meant we can just have local variable

`ObjectMapper objectMapper = new ObectMapper();`

Reply:
I see my bad.